### PR TITLE
Add swagger documentation to project

### LINKED ISSUE #64: Setup nestjs swagger
Should be enough for docs.

### PR DESCRIPTION
**Pull request by Satont**
fixes #64

### CODE REVIEW

File: apps/bot/src/api/app.module.ts [hunk 1]
```diff
@@ -8,11 +8,12 @@ import { AppService } from "./app.service";
...
-        ...(await getConnectionOptions()),
-        name: "microservice",
-        entities: Object.values(typeOrmEntities),
-      }),
+      useFactory: async () =>
+        ({
+          ...(await getConnectionOptions()),
+          name: "microservice",
+          entities: Object.values(typeOrmEntities),
+        } as any),
```
Comment (Satont) [inline apps/bot/src/api/app.module.ts:16]: Need to check does bot works correctly after workround.

Comment (LWJerri) [reply apps/bot/src/api/app.module.ts:16]: Hello. I can make it, but not now.

Comment (Satont) [reply apps/bot/src/api/app.module.ts:16]: Thank you but it's more like self todo.

Comment (Satont) [reply apps/bot/src/api/app.module.ts:16]: Working ok.

**Pull request merged**